### ISSUE TITLE
Hello, what is the value of the __init__ module noise and bound parameter in the python file of MAA2C? looking forward to your answer!

### ISSUE THREAD

**Issue by Yzhtc** [XinyaoQiu/DRL-for-edge-computing#6]
Hello, what is the value of the __init__ module noise and bound parameter in the python file of MAA2C?
****
    def __init__(self, env, n_agents, state_dim, action_dim, action_lower_bound, action_higher_bound,
                 noise, bound, memory_capacity=10, max_steps=None,
                 roll_out_n_steps=10, tau=300, 
                 reward_gamma=0.99, reward_scale=1., done_penalty=-10,
                 actor_output_act=nn.functional.softmax, critic_loss="huber",
                 actor_lr=0.01, critic_lr=0.01, training_strategy="centralized",
                 optimizer_type="rmsprop", entropy_reg=0.00,
                 max_grad_norm=None, batch_size=10, episodes_before_train=0,
                 use_cuda=False, actor_parameter_sharing=False, critic_parameter_sharing=False, 
                 epsilon_start=0.9, epsilon_end=0.05, epsilon_decay=100):

_Originally posted by @Yzhtc in https://github.com/XinyaoQiu/DRL-for-edge-computing/issues/5#issuecomment-1756595431_
